frontend/src/__tests__/unit/resourceEquality.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { isCpuLimitEqual, isMemoryLimitEqual } from '~/utilities/valueUnits';
```

Comment:
Thanks for adding unit tests!

We've recently moved to co-locating unit tests near their target source file. See https://github.com/opendatahub-io/odh-dashboard/pull/1773

Could you please rebase this PR and then relocate this test file to: `src/utilities/__tests__/valueUnits.spec.ts`

Reply:
sure